transport/httptransport/http_transport.go

```diff
@@ -292,15 +292,15 @@ func (t *transfer) execute(ctx context.Context) error {
 
 	// Check if the control file exists and create it if it doesn't. Control file captures the number of chunks that the transfer has been started with.
 	// If the number of chunks changes half way through, the transfer should continue with the same chunking setting.
-	nChunks := t.nChunks
+	chunks := t.nChunks
```

Comment:
i kinda like nChunks as it indicates a number instead of the chunks themselves.

Reply:
It was clashing with the constant name.

Reply:
Renamed the constant